Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "Apache-2.0 WITH LLVM-exception"
 rust-version = "1.67"
 
 [workspace.package]
-version = "1.2.0"
+version = "1.3.0-pre0"
```

Comment:
Should this be 1.2.1 or does it get overridden in CI anyway?

Reply:
It will get overwritten by https://github.com/fermyon/spin/pull/1519 (once i change the base to v1.2 after merging this)